DuckDuckGo/AppDelegate.swift

```diff
@@ -43,6 +43,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // MARK: lifecycle
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+                
+        if ProcessInfo.processInfo.environment.contains(where: { $0.key == "atb-testing" }) {
```

Comment:
Optional: Can we extract this block into self documenting methods something like `ifAtbTesting` for the boolean check and `resetStatisticsData` for processing if that's true.